src/test/shell/integration/runfiles_test.sh

```diff
@@ -179,14 +179,15 @@ EOF
 
   # that accounts for everything
   cd ../..
-  # For shell binary, we build both `bin` and `bin.exe`, but on Linux we only build `bin`
-  # That's why we have one more symlink on Windows.
+  # For shell binary and python binary, we build both `bin` and `bin.exe`,
+  # but on Linux we only build `bin`.
+  # That's why we have two more symlinks on Windows.
```

Comment:
Two more? But `bin.exe` is just one more.

Reply:
We have one more for shell binary and one more for python binary.